packages/view-util/test/index.test.js

```diff
@@ -0,0 +1,100 @@
+import { expect } from 'chai'
+import { ssz, testData } from '@view-builder/testing'
+import { view, viewBuilder } from '@view-builder/core/ns.js'
+import { rdf, schema, xsd } from '@tpluscode/rdf-ns-builders'
+import { prepareViewPointer } from '../index.js'
+
+describe('@view-builder/view-util', () => {
+  describe('prepareViewPointer', () => {
+    context('filter with simple operator', () => {
+      it('creates dimension for filter with deep path', async () => {
+        // given
+        const builderView = await testData`
+        <>
+          ${view.dimension} <#ZEIT> ;
+          ${view.filter} [
+            ${view.operation} ${view.Lt} ;
+            ${view.argument} "1950-01-01"^^${xsd.date} ;
+            ${viewBuilder.baseDimension} <#ZEIT> ;
+            ${viewBuilder.drillDownProperty} <https://ld.stadt-zuerich.ch/schema/hasEnd> ;
```

Comment:
I don't think we have drillDownProperty in rdf-cube-view-query

Reply:
This is view builder's namespace

Reply:
This becomes part of a SHACL path like `view:path ( <ZEIT> <hasEnd> )`

Reply:
Taking a step back. The point is that the view in view builder is not a complete `view:View` because it would be difficult to maintain its graph representation given the necessity to dynamically change it in multiple places.

Hence, some helper properties in the `view-builder:` namespace and a little post-processing code to make the view functional for the query lib and cube viewer. 

That code is now extracted to `packages/view-util`. It will be reused by the API when publishing views